pygmt/src/tilemap.py

```diff
@@ -34,86 +36,72 @@
+    provide Spherical Mercator (EPSG:3857) coordinates to the ``region`` parameter.
 
     {aliases}
 
     Parameters
     ----------
-    region : list
-        The bounding box of the map in the form of a list [*xmin*, *xmax*,
-        *ymin*, *ymax*]. These coordinates should be in longitude/latitude if
-        ``lonlat=True`` or Spherical Mercator (EPSG:3857) if ``lonlat=False``.
-
-    zoom : int or str
-        Optional. Level of detail. Higher levels (e.g. ``22``) mean a zoom
-        level closer to the Earth's surface, with more tiles covering a smaller
-        geographical area and thus more detail. Lower levels (e.g. ``0``) mean
-        a zoom level further from the Earth's surface, with less tiles covering
-        a larger geographical area and thus less detail [Default is
-        ``"auto"`` to automatically determine the zoom level based on the
-        bounding box region extent].
+    region
+        The bounding box of the map in the form of a list [*xmin*, *xmax*, *ymin*,
+        *ymax*]. These coordinates should be in longitude/latitude if ``lonlat=True`` or
+        Spherical Mercator (EPSG:3857) if ``lonlat=False``.
+    zoom
+        Level of detail. Higher levels (e.g. ``22``) mean a zoom level closer to the
+        Earth's surface, with more tiles covering a smaller geographical area and thus
+        more detail. Lower levels (e.g. ``0``) mean a zoom level further from the
+        Earth's surface, with less tiles covering a larger geographical area and thus
+        less detail. Default is ``"auto"`` to automatically determine the zoom level
+        based on the bounding box region extent.
 
         .. note::
            The maximum possible zoom level may be smaller than ``22``, and depends on
            what is supported by the chosen web tile provider source.
-
-    source : xyzservices.TileProvider or str
-        Optional. The tile source: web tile provider or path to a local file.
-        Provide either:
-
-        - A web tile provider in the form of a
-          :class:`xyzservices.TileProvider` object. See
-          :doc:`Contextily providers <contextily:providers_deepdive>` for a
-          list of tile providers [Default is
-          ``xyzservices.providers.OpenStreetMap.HOT``, i.e. OpenStreetMap
-          Humanitarian web tiles].
-        - A web tile provider in the form of a URL. The placeholders for the
-          XYZ in the URL need to be {{x}}, {{y}}, {{z}}, respectively. E.g.
+    source
+        The tile source: web tile provider or path to a local file. Provide either:
+
+        - A web tile provider in the form of a :class:`xyzservices.TileProvider` object.
+          See :doc:`Contextily providers <contextily:providers_deepdive>` for a list of
+          tile providers. Default is ``xyzservices.providers.OpenStreetMap.HOT``, i.e.
+          OpenStreetMap Humanitarian web tiles.
+        - A web tile provider in the form of a URL. The placeholders for the XYZ in the
```

Comment:
I have to use double curly brackets here ``{{x}}``, otherwise the `fmt_docstrings` decorator will report `KeyError` exception (see https://github.com/GenericMappingTools/pygmt/actions/runs/8225955536/job/22491725235?pr=3095).

The ``{{x}}`` is correctly shown as `{x}` on the website.